server/src/api/api.ts

```diff
@@ -79,6 +79,25 @@ const resolvers: QueryTypeResolver|MutationTypeResolver = {
             admin: context.user.admin
         };
     },
+    users: async (root, _args, _context): Promise<User[]> => {
+        // @ts-ignore
+        const {args, context} = fixArguments(root, _args, _context);
+        let searchObj = args.search;
+
+        // Restrict results to current user for non-admins
+        if (!context.user.admin) {
+            searchObj = {
```

Comment:
this is just a safeguard right? Clientside state wise, I feel like "current user" should be held in a field "user", not "users"

Reply:
Yes, client users the `user` query primarily but gotta protect against snooping users